### PR TITLE
Mobile_Menu Setup

### PR DESCRIPTION
**Pull request by Santosh-Konappanavar**
### Portfolio website milestone.
 
 * [ ]  Implemented mobile menu slid in and out page using javascript
 * [ ]  Implemented a dynamic hamburger icon to show changes during mobile-menu changes.
 * [ ]  Added Eslintrc to check javascript codes

### CODE REVIEW

**Review by t-yanick — CHANGES_REQUESTED** [review]
Hi @Santosh-Konappanavar , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Mobile menu functionality working as expected.
:heavy_check_mark: Descriptive Readme file with explicit steps to install the project depenecies locally.
:heavy_check_mark:  Good use of Eslint for JavaScript linters and all passing.

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ] Great job working on this milestone, but there's some minior issue. In your ReadMe file, in the section of **Usage**, you have not stated the command a user needs to run locally to ba able to run and use your project. Kindly add that.
- I did this request here because your ReadMe file does not display in Files Changed

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ichala — CHANGES_REQUESTED** [review]
Hi Student
Good job so far! 
👏👏👏 
You still need to work on some issues to go to the next project, but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- No Linter errors  ✔️ 


 ## Required Changes ♻️


- [ ] **_🛑  As Mentioned by the previous reviewer kindly update the **Usage**  Section in your readme file where you describe how to run you project locally 🙏_**





## Optional suggestions

- N/A

**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -157,10 +157,11 @@ Install this project with:
 
 ### Usage
```
Comment (iambenkis) [inline README.md:158]: - [ ] [**OPTIONAL**] Good job adding documentation of your project, however i will suggest you to use some techniques to make your README file more professional :+1:  
User "``" to highlight something 

### Example

`npx eslint .`

User 
"```bash"
your code ...
"```"

### Example

```bash
npx exlint .
```
What do you think about these tips? :smile:

**Review by iambenkis — APPROVED** [review]
Hi @Santosh-Konappanavar , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Linters are passing :heavy_check_mark: 
The project is running successfully :heavy_check_mark:  
PR is well descriptive :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@iambenkis** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Santosh-Konappanavar) [reply README.md:158]: thanks I will consider it

**Pull request merged**